packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -114,6 +114,49 @@ describe('RestServer (integration)', () => {
       .expect(200, content);
   });
 
+  it('allows static assets to be mounted on multiple paths', async () => {
+    const root = FIXTURES;
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+      },
+    });
+
+    server.static('/html-0', root);
+    server.static('/html-1', root);
+    const content = fs
+      .readFileSync(path.join(root, 'index.html'))
+      .toString('utf-8');
+    await createClientForHandler(server.requestHandler)
+      .get('/html-0/index.html')
+      .expect('Content-Type', /text\/html/)
+      .expect(200, content);
+    await createClientForHandler(server.requestHandler)
+      .get('/html-1/index.html')
+      .expect('Content-Type', /text\/html/)
+      .expect(200, content);
+  });
+
+  it('allows static assets to be mounted on the same path multiple times', async () => {
+    const root = FIXTURES;
+    const dirA = path.join(root, 'dir-a');
```

Comment:
Let's find a more descriptive directory name than `dir-a` please.

I think it's time to reorganize the `fixtures` directory, I am proposing the following structure:

```
fixtures
 - cert.pem
 - key.pem
 - pfx.pfx
 - assets
    - index.html # moved from fixtures/index.html
 - other-assets
    - index.html
```

In the long term, I think it would be best to leverage `TestSandbox` to create temporary asset directories for the tests, but I guess that's out of scope of this pull request.